auth/src/main/java/com/firebase/ui/auth/ui/email/EmailActivity.java

```diff
@@ -116,7 +119,10 @@ public void onNewUser(User user) {
 
         TextInputLayout emailLayout = findViewById(R.id.email_layout);
 
-        if (getFlowParams().allowNewEmailAccounts) {
+        List<AuthUI.IdpConfig> providerConfig = getFlowParams().providerInfo;
+        int emailIndex = providerConfig.indexOf(new AuthUI.IdpConfig.EmailBuilder().build());
```

Comment:
Not really since that's the customizable order used to display the providers in the picker activity. However, I've cleaned this up to manually find the config with a for loop which is clearer.